airbyte-integrations/connectors/source-salesforce/unit_tests/api_test.py

```diff
@@ -528,3 +528,59 @@ def test_convert_to_standard_instance(stream_config, stream_api):
     bulk_stream = generate_stream("Account", stream_config, stream_api)
     rest_stream = bulk_stream.get_standard_instance()
     assert isinstance(rest_stream, IncrementalSalesforceStream)
+
+
+def test_bulk_stream_paging(stream_config, stream_api_pk):
+    LastModifiedDate1 = "2022-10-01T00:00:00Z"
```

Comment:
Please do not capitalize variables and use snake case for non-class variables. We might have a stricter linter in the future that will ask you to change this :D

Reply:
fixed